sympy/functions/combinatorial/factorials.py

```diff
@@ -822,18 +822,37 @@ class binomial(CombinatorialFunction):
     integers and the value of the binomial can be computed in terms
     of factorials::
 
-    .. math:: \binom{n}{k} = \frac{n!}{k!(n-k)!}
+    .. math:: \binom{n}{k} = \frac{n!}{k!(n - k)!}
 
-    For arbitrary ``n`` and a nonnegative, integer ``k``, the following
-    definition can be used (and is useful when evaluating summations)::
+    For arbitrary ``n`` and integer ``k``, we use Newton's Generalized
+    Binomial Theorem::
 
     .. math:: \binom{n}{k} = \frac{ff(n, k)}{k!}
 
-    When ``k`` is negative but ``n - k`` is a non-negative integer then
-    the following identity can be used::
+    .. math:: (x+y)^n & =\sum_{k=0}^\infty \binom{n}{k} x^{n-k} y^k
+
+    Using the aforementioned theorem, one can interpret the binomial
+    coefficient ``binomial(n, k)`` as the coefficient of
+    ``x^k``, ``y^k``, ``x^(n - k)``, ``y^(n - k)`` in the Series
+    expansion of ``(x + y)^n``.
+
+    When ``k`` is negative and ``n - k`` is a non-negative integer
+    then the following identity is used before the coefficient is
+    calculated::
 
     .. math:: \binom{n}{k} = \binom{n}{n - k}
 
+    .. math:: \binom{n}{k} = \frac{ff(n, n - k)}{(n - k)!}
+
+    The binomial coefficient is zero in the following cases, provided
```

Comment:
here's the proof for that.
![Screen Shot 2020-04-23 at 1 08 46 PM](https://user-images.githubusercontent.com/28682735/80074885-789e7e00-8567-11ea-9fe7-05048b64fbbc.png)

Reply:
Also see [[3] Proposition 5.4, Daniel E. Loeb](https://core.ac.uk/download/pdf/82732883.pdf)